Kernel/Arch/x86_64/Processor.h

```diff
@@ -318,11 +318,6 @@ class Processor {
         return read_gs_ptr(__builtin_offsetof(Processor, m_in_irq));
     }
 
-    ALWAYS_INLINE static void restore_in_critical(u32 critical)
-    {
-        write_gs_ptr(__builtin_offsetof(Processor, m_in_critical), critical);
```

Comment:
Note to self: They used to do different things even after 9babb92a4b6, but then 0a02496f04d3a83ec0eb65635d39dff0b3fd4304 made them do the same thing.

Reply:
Ah thanks, that does clear up some confusion I had too.